tasks/human_eval_074.rs

```diff
@@ -9,7 +9,54 @@ use vstd::prelude::*;
 
 verus! {
 
-// TODO: Put your solution (the specification, implementation, and proof) to the task here
+fn total_match<'a>(lst1: Vec<&'a str>, lst2: Vec<&'a str>) -> (ret: Option<Vec<&'a str>>)
```

Comment:
It might be reasonable to require that each list is of length `<= u32.MAX`, so that you don't need to return an option.  That would then rule out a trivial implementation that simply returns `None` all of the time.

Reply:
then the length of each string in the list will also need to be constrained. I've instead required that `None` cannot be trivially returned

Reply:
Okay, that seems reasonable.